src/candid.ts

```diff
@@ -0,0 +1,163 @@
+    .map(({ value }) => value)
+    .filter(isValidEin));
+  return [...eins];
+};
+
+const candid: CommandModule = {
+  command: 'candid',
+  describe: 'Interact with the Candid Premier API',
+  builder: (y) => (y
+    .option('candid-api-key', {
+      describe: 'Candid Premier API key; get from account management at https://dashboard.candid.org/',
+      demandOption: true,
+      global: true,
+      type: 'string',
+    })
+    .command(
+      'lookup <ein>',
+      'Fetch and display information about an organization by its EIN',
+      {
+        'output-file': {
+          alias: 'write',
+          describe: 'Write organization information to the specified JSON file',
+          normalize: true,
+          type: 'string',
+        },
+      },
+      async (args) => {
+        const result = await getCandidProfile(
+          args.candidApiKey as string,
+          args.ein as string,
+        );
+
+        if (typeof args.outputFile === 'string') {
+          await writeFile(
+            args.outputFile,
+            JSON.stringify(result, null, 2),
+          );
+          logger.info(`Wrote Candid data for ${args.ein as string} to ${args.outputFile}`);
+        } else {
+          logger.info('Candid result', { result });
+        }
+      },
+    )
+    .command(
+      'update <ein>',
+      'Fetch information about an organization by its EIN, and upload to the PDC',
+      oidcOptions,
+      async (args) => {
+        const token = await getToken(
+          args.oidcBaseUrl as string,
+          args.oidcClientId as string,
+          args.oidcClientSecret as string,
+        );
+        const data = await getCandidProfile(args.candidApiKey as string, args.ein as string);
+        await postPlatformProviderData(
+          args.pdcApiBaseUrl as string,
+          token,
+          args.ein as string,
+          'Candid',
+          data,
+        );
+        logger.info(`Wrote data for ${args.ein as string} to PDC`);
+      },
+    )
+    .command(
+      'update-all',
+      'Update Candid profiles for all PDC proposals',
+      oidcOptions,
+      async (args) => {
+        const token = await getToken(
+          args.oidcBaseUrl as string,
+          args.oidcClientId as string,
+          args.oidcClientSecret as string,
+        );
+        const eins = await getEinsFromPdc(args.pdcApiBaseUrl as string, token);
+        logger.info(`Found ${eins.length} valid EINs to look up in Candid`);
+
+        for (let i = 0; i < eins.length; i += 1) {
+          /* eslint-disable no-await-in-loop -- rate limit ourselves so Candid doesn't */
+          const ein = eins[i];
+          if (typeof ein !== 'string') {
+            break;
+          }
+          const data = await getCandidProfile(args.candidApiKey as string, ein);
+          await postPlatformProviderData(
+            args.pdcApiBaseUrl as string,
+            token,
+            ein,
+            'Candid',
```

Comment:
It turns out the front end will be simpler if we make these URL-convenient values. @slifty, can you please use `charity-navigator` when you post your data?

```suggestion
            'candid',
```